### DISCUSSION TITLE
[HUGO] Generated css includes console.logs

### DISCUSSION THREAD

**Discussion by canvural** [tailwindlabs/tailwindcss#2196] [Help]
Hello,

I'm using the static site generator Hugo. And to build the css I'm using [Hugo pipeline with PostCSS](https://gohugo.io/hugo-pipes/postcss/).

My `postcss.config.js` file:

```js
const production = process.env.HUGO_ENVIRONMENT === "production";

module.exports = {
    plugins: [
        require("tailwindcss"),
        require("autoprefixer"),
        production && require("cssnano")
    ]
};

```

After running `hugo` command the generated  css file includes the console.logs for the feature flag stuff

```

info - You have opted-in to future-facing breaking changes: removeDeprecatedGapUtilities
info - These changes are stable and will be the default behavior in the next major version of Tailwind.
/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */html{line-height:1.15;
...
```

Using Tailwind 1.7.1

Did anybody encounter this type of issue before?

(I know this sounds like a problem with Hugo, but asking here also to reach more people!)

**Comment by brennanangel** [comment]
@canvural we had the same issue. 
[1.7.3 fixed this](https://github.com/tailwindlabs/tailwindcss/releases/tag/v1.7.3):
`Log feature flag notices to stderr instead of stdout to preserve compatibility with pipe-based build systems`
Works fine on Hugo/netlify for us now
